src/couch_replicator_scheduler.erl

```diff
@@ -279,13 +281,12 @@ start_jobs(Count, State) ->
     [start_job_int(Job, State) || Job <- Runnable3],
     ok.
 
-
-stop_jobs(Count, State) ->
-    Running0 = running_jobs(),
+-spec stop_jobs(non_neg_integer(), boolean(), #state{}) -> non_neg_integer().
+stop_jobs(Count, Continuous, State) ->
+    Running0 = running_jobs(Continuous),
     Running1 = lists:sort(fun oldest_job_first/2, Running0),
```

Comment:
`Running1 = lists:filter(fun not_continuous/1, Running0),` and adjust the variable names further down.